dotnet/Services/ProductReviewService.cs

```diff
@@ -413,13 +414,23 @@ public async Task<ReviewsResponseWrapper> GetReviewsByProductId(string productId
 
                 AppSettings settings = await GetAppSettings();
 
-                if (pastRevNLocale)
+                if (Locale.localeList.ContainsKey(locale))
```

Comment:
Need to check for null or empty locale
if (!string.IsNullOrEmpty(locale) && Locale.localeList.ContainsKey(locale))